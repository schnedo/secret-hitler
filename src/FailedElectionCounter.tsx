import { ReactElement } from "react";
import { ElectionRound } from "./game";

export interface FailedElectionCounterProps {
  electionCounter: ElectionRound;
}

export default function FailedElectionCounter({
  electionCounter,
}: FailedElectionCounterProps): ReactElement {
  return <div>electionRound {electionCounter}/3</div>;
}
