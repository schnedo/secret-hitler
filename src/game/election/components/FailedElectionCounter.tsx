import { ReactElement } from "react";
import { ElectionRound } from "..";

export interface FailedElectionCounterProps {
  electionCounter: ElectionRound;
}

export default function FailedElectionCounter({
  electionCounter,
}: FailedElectionCounterProps): ReactElement {
  return <div>electionRound {electionCounter}/3</div>;
}
