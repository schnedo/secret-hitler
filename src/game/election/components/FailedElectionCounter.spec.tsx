import { render } from "@testing-library/react";
import FailedElectionCounter from "./FailedElectionCounter";

it("should render correctly", async () => {
  expect.hasAssertions();

  const { container } = render(<FailedElectionCounter electionCounter={1} />);

  expect(container).toMatchSnapshot();
});
