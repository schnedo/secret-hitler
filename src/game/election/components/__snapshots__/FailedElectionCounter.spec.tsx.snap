// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div>
    electionRound 
    1
    /3
  </div>
</div>
`;
